app/services/clients/base_client/wrapper.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (method, verb) {
     const context = {
       correlationId: correlator.getId(),
       startTime: new Date(),
-      url: joinURL(lodash.get(opts, 'baseUrl', ''), opts.url),
+      url: joinURL(lodash.get(opts, 'baseUrl', ''), opts.uri),
```

Comment:
As it was explained to me this was changed because `opts.url` is never used/ set and this causes this dep bump for `url-join` to fail. 

Just making a note that to double check the logic leading up to this join and quickly search the repo for the wrapper `url` vs. `uri` use - once that's done this change should be fine.

Reply:
Looking further into this, it doesn't factor in what seems to be the most common case which is setting `baseUrl` and `url`. Higher level code expects this to then be put together into a single URL, for example:

```
{ 
baseUrl: 'http://mybaseurl.com',
url: '/products'
}
```

Given this case, the attribute `opts.uri` would not be defined and there `opts.uri` would never be assigned (given `if (uri && !opts.uri && !opts.url) opts.uri = uri`). 

Potentially it would be better to guard against empty `opts.url` strings - I can't think of a use case where this should ever be valid however.